### DISCUSSION TITLE
Welcome to PolyMath Discussions!

### DISCUSSION THREAD

**Discussion by SergeStinckwich** [PolyMathOrg/PolyMath#182] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hemalvarambhia** [comment]
Hi :wave: 

I'm Hemal, a contributor to PolyMath. I am a software engineer based in London (staff engineer at Simply Business). My training was in Physics at UCL (University College London).

Worth mentioning - there is also the PolyMath discord channel in the Pharo server.

**Comment by kbrus87** [comment]
How do I install this polymath on my local computer using windows?

**Reply by hemalvarambhia** [reply]
Do you have an Pharo Smalltalk image set up already? If so, then you will want to use Iceberg to clone the code from github.

**Reply by hemalvarambhia** [reply]
I recommend using Pharo Installer to set up an imagine because that tool makes it easy to do.
